src/components/Experiment/QuickInput/Tabs/SampleInput/SampleInputsTab.js

```diff
@@ -76,6 +86,15 @@ export default function SampleInputsTab(props) {
         );
     }
 
+    function makeSampleVideoInput(url, index) {
+        return (
+            <button onClick={() => { inputHandlerForPreview(url.src); selectInput(index); }} key={index} className={getElement(getInputClassName(url))}>
+                <video src={url.src} alt={url.alt} autoPlay muted={true} loop />
```

Comment:
Give this a `className={getElement("sample-audio-content")}` or something similar, and then use that in `SampleInputsTab.scss` instead of `audio { ... }`